terragrunt/aws/glue/etl/platform/gc_forms/gx/expectations/forms-template_suite.json

```
@@ -0,0 +1,243 @@
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "brand", "type_list": ["str", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "checkbox_count", "type_list": ["int", "int64", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "combobox_count", "type_list": ["int", "int64", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "dropdown_count", "type_list": ["int", "int64", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "dynamicrow_count", "type_list": ["int", "int64", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "fileinput_count", "type_list": ["int", "int64", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "formatteddate_count", "type_list": ["int", "int64", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "radio_count", "type_list": ["int", "int64", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "richtext_count", "type_list": ["int", "int64", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "textarea_count", "type_list": ["int", "int64", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "textfield_count", "type_list": ["int", "int64", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "addresscomplete_count", "type_list": ["int", "int64", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "year", "type_list": ["str", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_in_type_list",
+      "kwargs": { "column": "month", "type_list": ["str", "object"] },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_be_unique",
+      "kwargs": { "column": "id" },
+      "meta": {}
+    },
+    {
+      "expectation_type": "expect_column_values_to_match_regex",
+      "kwargs": {
+        "column": "id",
+        "regex": "^.{25}$"
```

I like this but unfortunately there are some cases where we have shorter `id` values.  We could tweak it like so:
```suggestion
        "regex": "^[a-b0-9]{1,25}$"
```